### PR TITLE
Added Horseman's Axe

### PR DESCRIPTION
**Pull request by Meowkov**
Draft, ready for balancing.

### CODE REVIEW

**Review by Yeldur — CHANGES_REQUESTED** [review]
After discussion with Meow we're not 100% sure how strong this is going to be but are leaning towards it being too powerful, obviously we can see the utilisation of exporter to leverage extremely low handling/speed because it won't impact a cav player in exchange for massive damage at a very low tier similar to Cata mace.

From discussing we'd like for one more horseman axe to be made, one that is more balanced in its stats (I.E. higher handling + speed), and this one can remain as it is. This way if the axe submitted is too strong, we have the ability to disable the OP one and shift it to a better balanced axe to mitigate any problems caused.

**Review by Meowkov — COMMENTED** [review]
Looking good, approved.

**Review by Yeldur — DISMISSED** [review]
Approving on Meow's behalf

**Review by Erdertus — APPROVED** [review]
Horse Man Saxe

File: items.json [hunk 1]
```diff
@@ -40113,7 +40113,7 @@
     "weight": 3.1,
     "tier": 7.56281948,
     "rank": 3,
-    "requirement": 15,
+    "requirement": 18,
```
Comment (Muparadzi) [inline items.json:40116]: Can you manually remove these changes to str requirement on the xbows? This should be fixed when #473 is merged.

Comment (Muparadzi) [inline items.json:40116]: Can you manually remove the change in str requirement in this PR? This should be fixed when #473 is merged.

**Review by Muparadzi — CHANGES_REQUESTED** [review]

File: items.json (hunk 1 above)
Comment (Meowkov) [reply items.json]: Done

**Review by Muparadzi — APPROVED** [review]

**Review by Yeldur — DISMISSED** [review]
After double checking things with Meow and testing myself, I'm happy - Approving

**Comment by Yeldur** [comment]
Rebase required following merge of https://github.com/namidaka/itembalancing/pull/476

**Review by Yeldur — APPROVED** [review]
rebase complete, approved

**Pull request merged**